packages/common/src/hooks/useListData/useSortBy.test.ts

```diff
@@ -53,9 +47,7 @@ describe('useSortBy', () => {
   });
 
   it('has the correct values after triggering a few sort bys in sequence', () => {
-    const { result } = renderHook(() =>
-      useSortBy<TestSortBy>({ key: 'id', isDesc: false })
-    );
+    const { result } = renderHook(() => useSortBy<TestSortBy>('id'));
```

Comment:
this is just the initial sort by so I thought that just passing the field was a good idea. I'm now realising it isn't, and that you will want to be able to specify descending or ascending differently for different columns